### ISSUE TITLE
No view is registered with id: sshHosts

### ISSUE THREAD

**Issue by sendaoYan** [VSCodium/vscodium#1912]
**Describe the bug**
vscodium report `No view is registered with id: sshHosts`

**Please confirm that this problem is VSCodium-specific**
- [ ] This bug doesn't happen if I use Microsoft's Visual Studio Code. It only happens in VSCodium.

**Please confirm that the issue/resolution isn't already documented**
- [ ] I checked [the Docs page](https://github.com/VSCodium/vscodium/blob/master/docs/index.md) and [the Troubleshooting page](https://github.com/VSCodium/vscodium/blob/master/docs/troubleshooting.md) and my issue is not mentioned there.

**To Reproduce**

**Expected behavior**

**Screenshots**

![image](https://github.com/VSCodium/vscodium/assets/24123821/6d7b0c45-69dd-4182-a6f8-eba5d0bd2202)

**Desktop (please complete the following information):**
 - OS: [e.g. Mac OS, Ubuntu 20.04, etc]Mac OS
 - Architecture [e.g. x64, ia32, arm64]aem64
 - Version [e.g. 1.33.0]
 - App Manager [e.g. Winget, Homebrew, Snap, AUR, RPM, Nix, ...]
 - Sandboxed [e.g. no, Flatpak, Snap]

**Additional context**

**Comment by zpiatt** [comment]
![Screenshot 2024-11-13 061515](https://github.com/user-attachments/assets/e7e9bacd-4f13-44dd-bebb-4f9ac9018e97)

Can confirm this is is present on Windows 11.

    OS: Windows 11 Pro 24H2
    Architecture: x64
    Version: 1.95.2.24313

**Comment by daiyam** [comment]
I assume it's with the extension `Open Remote - SSH`.
What's your actions?

**Comment by zpiatt** [comment]
Yes `Open Remote - SSH v0.0.47`. It appears to have only happened on initial connection after selecting a folder to open.

![Screenshot 2024-11-13 070912](https://github.com/user-attachments/assets/bcf0daa0-6658-4d68-adf5-3ef4b68c4a2c)

```
    onDidChangeNotification(i) {
        i.kind === 0 && (this.triggerAriaAlert(i.item),
        i.item.severity === st.Error && (i.item.message.original instanceof Error ? console.error(i.item.message.original) : console.error(dr(i.item.message.linkedText.toString(), !0))))
    }
```

**Comment by fredkumar89** [comment]
its still occuring on latest update
